src/inspector_js_api.cc

```diff
@@ -96,7 +97,7 @@ class JSBindingsConnection : public AsyncWrap {
   static void New(const FunctionCallbackInfo<Value>& info) {
     Environment* env = Environment::GetCurrent(info);
     if (!info[0]->IsFunction()) {
-      env->ThrowTypeError("Message callback is required");
+      env->THROW_ERR_INVALID_CALLBACK("Message callback is required");
```

Comment:
These should really use standard messages that could be specified in the macro definitions... e.g. this should be `env->THROW_ERR_INVALID_CALLBACK()` and use the same error message text as the javascript equivalent.

Reply:
@jasnell Yes, but this PR is more about assigning the codes in case we don't have the time to migrate them all before v10. Most of the errors touched here can be thrown in JS land anyway so I've left most of the messages as-is since if we are going to change them, might as well just do it in JS.

Reply:
Ok, that makes sense. I just wouldn't want it to fall off the radar... that is, I don't want folks to think that just because the code is assigned, there's nothing left to do on these.